src/excmds.ts

```diff
@@ -4108,14 +4113,21 @@ export async function tab_helper(interactive: boolean, anyWindow: boolean, ...ke
  *  - [[comclear]]
  */
 //#background
-export function command(name: string, ...definition: string[]) {
+export async function command(...args: string[]) {
     // Test if alias creates an alias loop.
+    let url = null
+    if (args[0] === "--url") {
+        args.shift()
+        url = args.shift()
+    }
+    const name = args[0]
+    const def = args.slice(1).join(" ")
     try {
-        const def = definition.join(" ")
+        if (url) await config.setURL(url, "exaliases", name, def)
+        else await config.set("exaliases", name, def)
         aliases.expandExstr(name)
-        return config.set("exaliases", name, def)
     } catch (e) {
-        config.unset("exaliases", name)
+        await config.unset("exaliases", name)
```

Comment:
as i understand it in this code we: 
1. set the alias in the config
2. check that there are no loops (throw an error)
3. catch that error and unset the alias

so surely here there needs to be a matching unsetURL?